connectors/src/lib/data_sources.ts

```diff
@@ -153,6 +153,7 @@ async function _upsertToDatasource({
         title,
         mime_type: mimeType,
         tags: tags?.map((tag) => tag.substring(0, 512)),
+        parent_id: null,
```

Comment:
do we send null even if parents[1] is set?

Reply:
we could, but I think we should first ensure parents are in correct format and order for all connectors ?

Reply:
Ok, would that make sense then to store that in a card, with all other things we can do once parents are properly standardized?
Or maybe too small for a card but a way to remember?

Reply:
we can add a comment in all "Standardize parents" cards

Reply:
as discussed IRL, we can directly pass `parents[1] ?? null` here

Reply:
done